features/kata/operator.feature

```diff
@@ -0,0 +1,12 @@
+Feature: kata operator related tests
+  Background:
+  Given valid cluster type for kata tests exists
+
+  # @author valiev@redhat.com
+  # @case_id OCP-41813
+  @admin
+  @destructive
+  Scenario: operator successfully installed
+    Given catalogsource "redhat-operators" exists in "openshift-marketplace" namespace
```

Comment:
@valtair1891, @cpmeadors  I have recommend[ multiple times](https://github.com/openshift/verification-tests/pull/2442#issuecomment-962016250) to consolidate all three steps into one perhaps something like `Given kata operator is installed successfully`.  Like a function, a cucumber step should hide the details on the implementation.  The advantage is so that in case the installation workflow is different, then your feature file does not need to be changed, only top level step needs to be modified.

Reply:
@pruan-rht Done
and tested
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/kata-ci/job/4_9_Sandboxed_Operator_Installation_OpenShift/288/console